### ISSUE TITLE
Add Parser get_parameters method

### ISSUE THREAD

**Issue by awig** [ttinoco/PFNET#177]
@ttinoco  What do you think about adding a get_parameters method to Parser class? It would be nice to see in Python (or Julia) what the current parameters ('keep_line_outages' etc.) are set to.

**Comment by ttinoco** [comment]
I agree, let do this update at the same time we change parser "set" to
parser "set_parameter" (singular)
